jobs/contact-retrieve-transcript/tests/unit/exportTranscript.test.ts

```diff
@@ -19,8 +20,39 @@ const messageList = [
     type: 'message',
     media: null,
   },
+  {
+    sid: 3,
+    dateCreated: 'blah',
+    from: 'counselor',
+    body: 'How are you today?',
+    index: 2,
+    type: 'message',
+    media: null,
+  },
 ];
 
+const childUserData = {
+  identity: 'child',
+  friendlyName: 'Child',
+  roleSid: 'childRoleSid',
+};
+
+const counselorUserData = {
+  identity: 'counselor',
+  friendlyName: 'Counselor',
+  roleSid: 'counselorRoleSid',
+};
+
+const childRoleData = {
+  sid: 'childRoleSid',
+  friendlyName: 'Child',
+};
+
+const counselorRoleData = {
+  sid: 'counselorRoleSid',
+  friendlyName: 'Counselor',
+};
+
 jest.mock('@tech-matters/hrm-twilio-client', () => {
```

Comment:
Eventually, the mock client in `@tech-matters/hrm-twilio-client` should probably be configurable so we don't have to duplicate it in tests, but for now this was the fastest path forward.